pymysql/parsers.py

```diff
@@ -0,0 +1,20 @@
+from ._compat import PY2
+
+if PY2:
+    import ConfigParser as configparser
+else:
+    import configparser
+
+
+class DefaultFileParser(configparser.RawConfigParser):
+
```

Comment:
You're right.
I haven't dropped 2.6 support yet.
I think it should be happen in PyMySQL 0.7...